python/lsst/pex/config/config.py

```diff
@@ -1045,13 +1053,19 @@ def saveToStream(self, outfile, root="config"):
         tmp = self._name
         self._rename(root)
         try:
+            self.collectImports()
+            # Remove self from the set, as it is handled explicitly below
+            self._imports.remove(self.__module__)
             configType = type(self)
             typeString = _typeStr(configType)
             outfile.write(u"import {}\n".format(configType.__module__))
             outfile.write(u"assert type({})=={}, 'config is of type %s.%s ".format(root, typeString))
             outfile.write(u"instead of {}' % (type({}).__module__, type({}).__name__)\n".format(typeString,
                                                                                                 root,
                                                                                                 root))
+            for imp in self._imports:
+                if imp in sys.modules and sys.modules[imp] is not None:
```

Comment:
Is there any way a module could actually be in `self._imports` and not `sys.modules`?

Reply:
Im not 100% sure, that bit of code was moved from below, but I think it may be possible to get something like that if someone uses importlib to lookup modules.